src/main.rs

```diff
@@ -17,6 +19,9 @@ fn main() {
     // writers::file_writer
     writers::file_writer::write(&toml_reader.toml_data, dir_builder).unwrap();
 
+    // format all .rs files in generated directory
+    writers::post_operator::format_all_rs(Path::new(&String::from("./GENERATED"))).unwrap();
```

Comment:
cleaner call could involve passing in the `&String` and generating a path on the method